x/oracle/types/event.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+const (
+	AttributeKeyRegisterOracle   = "oracle"
```

Comment:
`AttributeKeyOracle` might be better. Or `AttributeKeyOracleAddress`.

Reply:
I agree. :)